public/app/features/alerting/unified/components/AlertLabels.tsx

```diff
@@ -33,9 +34,28 @@ export const AlertLabels = ({ labels, commonLabels = {}, size }: Props) => {
 
   return (
     <div className={styles.wrapper} role="list" aria-label="Labels">
-      {labelsToShow.map(([label, value]) => (
-        <Label key={label + value} size={size} label={label} value={value} color={getLabelColor(label)} />
-      ))}
+      {labelsToShow.map(([label, value]) => {
```

Comment:
wrap the Label with a new div to make the label clickable only when `onLabelClick` prop is present.

Reply:
Do you think it's worth putting this into the `Label` component instead?

Reply:
I think you're right, as we want to enhance labels, and the reused component is Labels, not AlertLabel 👍